### PR TITLE
feat: Remove connect from Connect plugin build filename

### PR DESCRIPTION
**Pull request by henriknorin-ftrack**
Resolves : 

* CLICKUP-
* FT-e0209055-757a-4fe9-845c-e592236355de
* SENTRY-
* ZENDESK-

- [ ] I have added automatic tests where applicable.
- [ ] The PR contains a description of what has been changed.
- [ ] The description contains manual test instructions.
- [ ] The PR contains update to the release notes.
- [ ] The PR contains update to the documentation.

This PR has been tested on :

- [ ] Windows.
- [X] MacOs.
- [ ] Linux.

## Changes

- Remove connect from Connect plugin build filename
- Allow up to Python 3.11 for publisher and timetracker widget
- Fixed missing import in timetracker widget hook

## Test

Build and test publisher and timetracker.

### CODE REVIEW

File: tools/build.py [hunk 1]
```diff
@@ -485,9 +486,7 @@ def build_connect_plugin(args):
                     )
 
         logging.info('Creating archive')
-        archive_path = os.path.join(
-            BUILD_PATH, '{0}-{1}'.format(PROJECT_NAME, CONNECT_PLUGIN_VERSION)
-        )
+        archive_path = os.path.join(BUILD_PATH, os.path.basename(STAGING_PATH))
```
Comment (lluisFtrack) [inline tools/build.py:489]: Don't we need the connect_plugin_version? I see that is left unused just for logs in the code then?

Comment (henriknorin-ftrack) [reply tools/build.py:489]: The version number will come from the staging path, so it is preserved. Just wanted to make sure ZIP and the build folder names are 100% aligned

Comment (lluisFtrack) [reply tools/build.py:489]: Worth removing the CONNECT_PLUGIN_VERSION then as it seems that is not used in the file anymore?

Comment (henriknorin-ftrack) [reply tools/build.py:489]: Yes, this is just logged now.

What we should perhaps do in CI is check that the plugin version in __version__.py aligns with the tag, so you do no mistakes when creating the manual release? Same goes for libraries.

**Review by lluisFtrack — APPROVED** [review]

**Pull request merged**